environment/src/main/java/com/sequenceiq/environment/platformresource/PlatformParameterService.java

```diff
@@ -152,9 +154,17 @@ public PlatformResourceRequest getPlatformResourceRequest(
         PlatformResourceRequest platformResourceRequest = new PlatformResourceRequest();
 
         if (!Strings.isNullOrEmpty(credentialName)) {
-            platformResourceRequest.setCredential(credentialService.getByNameForAccountId(credentialName, accountId, ENVIRONMENT));
+            Credential credential = credentialService.getByNameForAccountId(credentialName, accountId, ENVIRONMENT);
```

Comment:
Can you do in the same request? filter for all type or something

Reply:
I mean, in the same sql query

Reply:
when we implemented this feature we decided to follow the approach that only one type can passed to the repository